setup.py

```diff
@@ -52,6 +52,7 @@
     "importlib_metadata;python_version<'3.8'",
     "azure-iot-device~=2.11",
     "tqdm~=4.62",
+    "treelib",
```

Comment:
Make sure to add the dependency licensing info to the thirdpartynotice https://github.com/Azure/azure-iot-cli-extension/blob/dev/thirdpartynotice